content/blog/2018-03-13-weathercan.md

```diff
@@ -0,0 +1,500 @@
+
+``` r
+# Data manipulation and plotting
+library(dplyr)
+library(ggplot2)
+
+# Checking data completeness
+library(naniar)
+
+# Access to data containing feeder visits by birds
+library(feedr)
+
+# Spatial analyses
+library(sf)
+library(mapview)
+```
+
+Well, I've told you it's easy to get data from `weathercan`, so let's start by doing so. For example, if you wanted to download weather data for all of Manitoba, Canada since the New Year, you have only three steps:
+
+1.  Load the package:
+
+    ``` r
+    library(weathercan)
+    ```
+
+2.  Look at the built in `stations` data set to find the specific stations you're interested in (you can also use the `stations_search()` function). Here, we'll use the [`dplyr`](http://dplyr.tidyverse.org/) package (part of [`tidyverse`](https://www.tidyverse.org)) to `filter()` stations to only those in the province of Manitoba, at daily intervals, and which have an end date of at least 2018 (which likely means it's still operational). Note that we'll also be removing some columns (`prov`, `climate_id`, `WMO_id`, `TC_id`) just for clarity.
+
+    ``` r
+    mb <- filter(stations, 
+                 prov == "MB",
+                 interval == "day",
+                 end >= 2018) %>%
+      select(-prov, -climate_id, -WMO_id, -TC_id)
+    mb
+    ```
+
+        ## # A tibble: 70 x 8
+        ##    station_name         station_id   lat    lon  elev interval start   end
+        ##    <chr>                <fct>      <dbl>  <dbl> <dbl> <chr>    <int> <int>
+        ##  1 BALDUR               3463        49.3 - 99.3   450 day       1962  2018
+        ##  2 BRANDON A            50821       49.9 -100.0   409 day       2012  2018
+        ##  3 BRANDON RCS          49909       49.9 -100.0   409 day       2012  2018
+        ##  4 CARBERRY CS          27741       49.9 - 99.4   384 day       1994  2018
+        ##  5 CYPRESS RIVER RCS    48128       49.6 - 99.1   374 day       2009  2018
+        ##  6 ELKHORN 2 EAST       3460        49.9 -101     498 day       1987  2018
+        ##  7 PORTAGE ROMANCE      45987       50.0 - 98.3   262 day       2007  2018
+        ##  8 PORTAGE LA PRAIRIE … 3519        50.0 - 98.3   259 day       1970  2018
+        ##  9 PORTAGE SOUTHPORT    10884       49.9 - 98.3   273 day       1992  2018
+        ## 10 ROBLIN               27119       51.2 -101     540 day       1996  2018
+        ## # ... with 60 more rows
+
+3.  Download all the data from the start of the year for these stations
+
+    ``` r
+    mb_weather_all <- weather_dl(station_ids = mb$station_id, 
+                                 start = "2018-01-01", 
+                                 interval = "day", quiet = TRUE)
+    ```
+
+A simple example
+----------------
+
+Where things get tricky are in the specific use cases. For example, what if you have multiple sites and multiple days for which you want a measure of temperature and precipitation?
```

Comment:
You could start this para like this: "What if you have multiple sites and multiple days for which you want a measure of temperature and precipitation?" and it sounds stronger without the "...things get tricky..."

Reply:
Start this section with a clear statement of what you will do / accomplish (i.e. you're going to combine weather and stream data from some stations near a site) in the example so reader doesn't get lost.